Description: Disable tests relying on external data
Forwarded: not-needed
Author: Antonin Delpeuch <antonin@delpeuch.eu>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -3601,6 +3601,7 @@ mod test {
     /// Be sure to check out the CSL
     /// [styles](https://github.com/citation-style-language/styles) repository
     /// into a sibling folder to run this test.
+    #[ignore]
     #[test]
     fn roundtrip_cbor_all() {
         fs::create_dir_all("tests/artifacts/styles").unwrap();
@@ -3642,6 +3643,7 @@ mod test {
     /// Be sure to check out the CSL
     /// [locales](https://github.com/citation-style-language/locales) repository
     /// into a sibling folder to run this test.
+    #[ignore]
     #[test]
     fn roundtrip_cbor_all_locales() {
         fs::create_dir_all("tests/artifacts/locales").unwrap();
